blinkit/page-2.html

```diff
@@ -0,0 +1,191 @@
+        <div id="delivery-time">Delivery in 10 mins</div>
+        <input class="nosubmit" type="search" placeholder="Search...">
+        <div class="btn-navbar">
+
+            <div id="Login"><a href="#">Login</a></div>
+            <button id="my-cart">My Cart</button>
+        </div>
+    </nav>
+    <section class="menu-1">
+        <ul class="menu-1-list">
+            <div>
+                <li>Vegetables and Fruits</li>
+            </div>
+            <div>
+                <li>Dairy and Breakfast</li>
+            </div>
+            <div>
+                <li>Munchies</li>
+            </div>
+            <div>
+                <li>Cold Drinks & Juices</li>
+            </div>
+            <div>
+                <li>Tea, Coffee & Health Drinks</li>
+            </div>
+            <div>
+                <li>Instant & Frozen Food</li>
+            </div>
+            <div>
+                <li>Bakery & Biscuits</li>
+            </div>
+            <div>
+                <li>More</li>
+            </div>
+        </ul>
+
+    </section>
+    <section class="place-order">
+        <div class="place-order-header">
+            <p>Place Order </p>
+        </div>
+        <br>
+        <br>
+        <div class="time-to-deliver">
+            <p>Delivery in 10 minutes</p>
+            <p>6 items</p>
+        </div>
+
+        <div class="items-added-to-cart">
+            <div class="item-added-to-cart">
+                <div class="item-detail-cart">
+                    <div class="item-image"><img src="images/391305a.jpg"></div>
+                    <div class="item-purchase-details">
+                        <p>New Potato</p>
+                        <p>1kg</p>
+                        <p>₹15 <span style="text-decoration: line-through;font-size: 14px; color: grey">₹17</span></p>
+                    </div>
+                </div>
+                <div class="quantity-selector-cart">
+                    <div>-</div>
+                    <div>2</div>
+                    <div>+</div>
+                </div>
+            </div>
+            <div class="items-added-to-cart">
+                <div class="item-added-to-cart">
+                    <div class="item-detail-cart">
+                        <div class="item-image"><img src="images/23383a.jpg"></div>
+                        <div class="item-purchase-details">
+                            <p>Local Carrot</p>
+                            <p>500g</p>
+                            <p>₹17 <span style="text-decoration: line-through; color: grey;">₹19</span></p>
+                        </div>
+                    </div>
+                    <div class="quantity-selector-cart">
+                        <div>-</div>
+                        <div>3</div>
+                        <div>+</div>
+                    </div>
+                </div>
+            </div>
+            <div class="items-added-to-cart">
+                <div class="item-added-to-cart">
```

Comment:
Do we really need this div?
We should not unnecessarily nest divs, unless its required.